src/ui/edit/wire-canvas-item.c

```diff
@@ -394,11 +394,9 @@ popup_helper (gpointer user_data)
 {
   BtEventIdleData *data = (BtEventIdleData *) user_data;
   BtWireCanvasItem *self = data->self;
-  guint32 activate_time = data->activate_time;
   FREE_EVENT_IDLE_DATA (data);
 
-  gtk_menu_popup (self->priv->context_menu, NULL, NULL, NULL, NULL,
-      GDK_BUTTON_SECONDARY, activate_time);
+  gtk_menu_popup_at_pointer (self->priv->context_menu, NULL);
   return FALSE;
```

Comment:
this is in the wrong commit

Reply:
Yes, sorry. The PR is going to have to be organised properly, it's too messy.